app/setup.py

```diff
@@ -29,20 +29,22 @@
 from app.keys import KEY_PURPOSE_SUBMISSION
 from app.new_relic import setup_newrelic
 from app.secrets import SecretStore, validate_required_secrets
+from app.talisman import Talisman
 
 from app.submitter.submitter import LogSubmitter, RabbitMQSubmitter
 
 import boto3
 from botocore.config import Config
 
 
-SECURE_HEADERS = {
+CACHE_HEADERS = {
     'Cache-Control': 'no-cache, no-store, must-revalidate',
     'Pragma': 'no-cache',
-    'Strict-Transport-Security': 'max-age=31536000; includeSubdomains',
-    'X-Frame-Options': 'DENY',
-    'X-Xss-Protection': '1; mode=block',
-    'X-Content-Type-Options': 'nosniff',
+}
+
+CSP_POLICY = {
+    'default-src': ["'self'", ],
```

Comment:
You will also need to allow google analytics as we run those scripts and it seems like its violates the policy if not added

Reply:
Ah yeah. strict-dynamic should allow GA to work but not all browsers support it.

I'm thinking that we should move to a whitelist TBH.

Reply:
And i guess we would want to add the CDN to this too, but we will do that when we integrate it.